jbpm-xes/pom.xml

```diff
@@ -27,13 +27,29 @@
   <name>jBPM :: XES Exporter</name>
   <description>jBPM XES Exporter</description>
   <artifactId>jbpm-xes</artifactId>
+  
+  <properties>
+    <version.commons-cli>1.4</version.commons-cli>
+    <version.com.h2database>1.4.193</version.com.h2database>
+  </properties>
+   
 
   <dependencies>
 
     <dependency>
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-test</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
Just curious, why is this exclusion needed?

Reply:
It is needed to avoid duplicated classes with commons-logging:

  Found in:
    commons-logging:commons-logging:jar:1.1.1:compile
    org.slf4j:jcl-over-slf4j:jar:1.7.25:test
  Duplicate classes:
    org/apache/commons/logging/impl/SimpleLog$1.class
    org/apache/commons/logging/Log.class
    org/apache/commons/logging/LogConfigurationException.class
    org/apache/commons/logging/impl/NoOpLog.class
    org/apache/commons/logging/impl/SimpleLog.class
    org/apache/commons/logging/LogFactory.class